tests/test_vqe.py

```diff
@@ -591,7 +591,7 @@ def test_metric_tensor(self, approx):
         """Test that the metric tensor can be calculated."""
 
         dev = qml.device("default.qubit", wires=3)
-        p = np.array([1.0, 1.0, 1.0])
+        p = pnp.array([1.0, 1.0, 1.0], requires_grad=True)
```

Comment:
how come this test was altered?

Reply:
To be completely honest - I don't know why it was not failing before.

Reply:
😆 the classic 'the test _should_ have been failing'